test/reactTags.test.js

```diff
@@ -201,6 +201,21 @@ test("should not set any property of this.textInput when readOnly", () => {
   $tag.simulate("click");
 });
 
+test("accepts non-internal props on the top-level div", () => {
```

Comment:
should accept non-internal props on the top-level div ?

Reply:
Ok.